variables.tf

```diff
@@ -101,6 +101,12 @@ variable "provisionopenvpn_policy_name" {
   default     = "ProvisionOpenVPN"
 }
 
+variable "root_disk_size" {
+  type        = number
+  description = "The size of the OpenVPN instance's root disk in GiB."
+  default     = 8
```

Comment:
Do we care about ensuring a minimum size?

Reply:
What exactly do you mean by that @mcdonnnj?